server/src/main/java/org/elasticsearch/action/admin/indices/validate/query/ValidateQueryResponse.java

```diff
@@ -36,38 +27,10 @@ public class ValidateQueryResponse extends BroadcastResponse {
     public static final String VALID_FIELD = "valid";
     public static final String EXPLANATIONS_FIELD = "explanations";
 
-    @SuppressWarnings("unchecked")
-    static final ConstructingObjectParser<ValidateQueryResponse, Void> PARSER = new ConstructingObjectParser<>(
-        "validate_query",
-        true,
-        arg -> {
-            BaseBroadcastResponse response = (BaseBroadcastResponse) arg[0];
-            return new ValidateQueryResponse(
-                (boolean) arg[1],
-                (List<QueryExplanation>) arg[2],
-                response.getTotalShards(),
-                response.getSuccessfulShards(),
-                response.getFailedShards(),
-                Arrays.asList(response.getShardFailures())
-            );
-        }
-    );
-    static {
-        declareBroadcastFields(PARSER);
```

Comment:
Looks like `TermsEnumResponse.PARSER` is another candidate, and with that `declareBroadcastFields` also becomes test-only.

Reply:
Right :) There's endless candidates like this I just eventually put a lid on this to not blow up the PR size so much. Maybe do a follow-up for this (and a couple others more)?